src/main/java/org/opensearchmetrics/metrics/release/ReleaseIssueChecker.java

```diff
@@ -0,0 +1,61 @@
+package org.opensearchmetrics.metrics.release;
+
+import org.opensearch.action.search.SearchRequest;
+import org.opensearch.action.search.SearchResponse;
+import org.opensearch.index.query.BoolQueryBuilder;
+import org.opensearch.index.query.QueryBuilders;
+import org.opensearch.search.SearchHit;
+import org.opensearch.search.aggregations.AggregationBuilders;
+import org.opensearch.search.aggregations.bucket.terms.Terms;
+import org.opensearch.search.builder.SearchSourceBuilder;
+import org.opensearchmetrics.util.OpenSearchUtil;
+
+import javax.inject.Inject;
+import java.util.Arrays;
+
+public class ReleaseIssueChecker {
+
+    @Inject
+    public ReleaseIssueChecker() {}
+
+    public String[] releaseOwners(String releaseVersion, String repo, OpenSearchUtil openSearchUtil) {
+        BoolQueryBuilder boolQueryBuilder = QueryBuilders.boolQuery();
+        boolQueryBuilder.must(QueryBuilders.matchQuery("repository.keyword", repo));
+        boolQueryBuilder.must(QueryBuilders.matchQuery("title.keyword", "[RELEASE] Release version " + releaseVersion));
+        boolQueryBuilder.must(QueryBuilders.matchQuery("issue_pull_request", false));
+        SearchRequest searchRequest = new SearchRequest("github_issues");
+        SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder();
+        searchSourceBuilder.query(boolQueryBuilder);
+        searchSourceBuilder.size(9000);
+        searchSourceBuilder.aggregation(
+                AggregationBuilders.terms("issue_assignees")
```

Comment:
So we do still needs to rely on release team assign a owner to the issue before you can retrieve, or it will show as empty right?

Reply:
Thats true peter, coming from https://github.com/opensearch-project/opensearch-metrics/issues/61 the long term plan is to have the user mapping of GitHub to slack and tag in slack channel as well, but based on the discussion in the issue we can start with the GitHub assignee as entry criteria to ensure we have the owner in the GitHub representing the repo. So with now release owner the field will be empty and the field type is String[] so can accommodate more than one release owner as well (which is allowed by GitHub as well).